assets/styl/_modules/_form.styl

```diff
@@ -0,0 +1,59 @@
+// Forms widgets
+//
+// Colette provides styles for:
+// - Fields descriptions (with valid / invalid color indication)
+// - Valid / Invalid fields
+// - Also a special style for a flex parent with 3 children (label, infolink, input 100%)
+//
+// Note: you can also use Alerts components to handle success or errors messages.
+//
+// Weight: 2
+//
+// Markup: form.twig
+//
+// Styleguide: Components.Forms.Form
+.form
+    &-field-widget
+        // display a widget for form element that must be filled with text content
+        // e.g. input[text|password|email|url]
+
+        label
+            display block
+
+        ul
+            margin 0
+
+        //special style for a flex parent with 3 children (label, infolink, input 100%)
+        &-flex3
+            display flex
+            flex-wrap wrap
+
+            label
+                flex 1 auto
```

Comment:
do not use shorthand rules without all parameters for `flex` (because of some browser default value can currently be different)